doc/users/license.rst

```diff
@@ -12,7 +12,7 @@ licenses. Non-BSD compatible licenses (e.g., LGPL) are acceptable in
 matplotlib toolkits.  For a discussion of the motivations behind the
 licencing choice, see :ref:`license-discussion`.
 
-Copyright Policy
+Copyright policy
```

Comment:
Possibly a proper noun, but not sure.

Reply:
Seems to be used lowercase in plain text, e.g.

https://www.rightsdirect.com/10-tips-creating-copyright-policy/
http://www.copyright.com/learn/6-tips-creating-effective-copyright-policy/